lib/args.js

```diff
@@ -77,6 +76,10 @@ function args(argv) {
     opts['web-security'] = opts['web-security'] === 'true';
   }
 
+  if (opts.hasOwnProperty('https-server') && !opts['https-server']) {
+    opts['https-server'] = null;
```

Comment:
I think this is really not pretty. Is there another nicer way of saying "option yes, value no"?

Reply:
It's always going to be ugly when a variable is used for control flow and should also support a default value. Some refactoring of this could lead to splitting the two concerns, one variable to specify whether or not to start the server and one variable to specify which port to use.

Reply:
I'd try to not break the api for as long as possible.

There's nothing stopping us from creating more than one internal variable from just one CLI argument.

```js
var startServer = // was the https-server flag used?
var passedPort = // what was the value passed?
var tryOtherPorts = !passedPort;
var serverPort = tryOtherPorts ? defaultPort : passedPort;
```